### ISSUE TITLE
[BUG] No configuration button

### ISSUE THREAD

**Issue by Aoz0ra** [n4n0GH/hello#80]
**< The bug >**
There's no button to configure the hello shaders.

**< Steps to reproduce >**
System Settings > Workspace Behavior > Desktop Effects > hello

**< My expectations >**
There should be a configuration button, given that there's already a .kcfg file for it.

**< Bug inheritance >**
*If the function or feature is also present on KDE's own Breeze theme, does the same bug occur there as well?*
Not applicable.

**< Screenshot >**
![Screenshot_20200514_114306](https://user-images.githubusercontent.com/64323569/81955468-1ff75980-95d8-11ea-952c-609c585c2953.png)

**< Additional context >**
- Ubuntu 20.04 LTS “Focal Fossa”
- I believe I'm on the latest Linux and KDE as of writing
- Other effects that have configuration windows show the config buttons, so it's not my themes

**Comment by n4n0GH** [comment]
Yeah I kinda stopped working on that when I put the entire project on halt. Would be dope to have a settings menu there tho. Maybe someday.
